ballista/core/src/utils.rs

```diff
@@ -148,6 +150,20 @@ impl BallistaObjectStoreRegistry {
             }
         }
 
+        #[cfg(feature = "gcs")]
+        {
+            if url.to_string().starts_with("gs://") {
```

Comment:
Thanks @haoxins 👏
The current google cloud storage has 2 types of url schema: "gs://" or "gcs://". So here we also need to add support for "gcs://".

Reply:
done